Rakefile

```diff
@@ -74,14 +74,7 @@ task :new_cop, [:cop] do |_task, args|
   generator.write_source
   generator.write_spec
   generator.inject_require(root_file_path: 'lib/rubocop/cop/rspec_cops.rb')
-  generator.inject_config(config_file_path: 'config/default.yml',
-                          version_added: bump_minor_version)
```

Comment:
The requirements can be satisfied by default arguments as follows:

https://github.com/rubocop/rubocop/blob/bc4ae3ef798c8db4f7427369777deb69be8f4d63/lib/rubocop/cop/generator.rb#L133-L134
```ruby
      def inject_config(config_file_path: 'config/default.yml',
                        version_added: '<<next>>')
```